gemhardware/src/common/glib/GLIBMonitor.cc

```diff
@@ -313,12 +384,201 @@ void gem::hw::glib::GLIBMonitor::buildMonitorPage(xgi::Output* out)
+      std::string regname = "CLUSTER_SIZE";
+      for (int j = 0; j < 8; ++j) {
+        std::stringstream specregname;
+        specregname << regname << "_" << j << "_CNT";
+        *out << "<tr>"    << std::endl
+             << "<td>"    << std::endl
+             << specregname.str()
+             << "</td>"   << std::endl;
+
+        for (int i = 0; i < 12; ++i) {
+          INFO("GLIBMonitor::buildTriggerStatusTable creating OH" << i << " table header");
+          *out << "<td id=\"" << monpair->second.infoSpace->name() << "-OH" << i << "_" << specregname.str() << "\">" << std::endl
+               << "N/A"
+               << "</td>"   << std::endl;
+        }
+        *out << "<td>"    << std::endl
+             << "add"
+             << "</td>"   << std::endl
+             << "<td>"    << std::endl
+             << "desc"
+             << "</td>"   << std::endl
+             << "</tr>"   << std::endl;
+      }
+
+      regname = "CLUSTER_SIZE";
+      for (int j = 0; j < 8; ++j) {
+        std::stringstream specregname;
+        specregname << regname << "_" << j << "_RATE";
+        *out << "<tr>"    << std::endl
+             << "<td>"    << std::endl
+             << specregname.str()
+             << "</td>"   << std::endl;
+
+        for (int i = 0; i < 12; ++i) {
+          INFO("GLIBMonitor::buildTriggerStatusTable creating OH" << i << " table header");
+          *out << "<td id=\"" << monpair->second.infoSpace->name() << "-OH" << i << "_" << specregname.str() << "\">" << std::endl
+               << "N/A"
+               << "</td>"   << std::endl;
+        }
+        *out << "<td>"    << std::endl
+             << "add"
+             << "</td>"   << std::endl
+             << "<td>"    << std::endl
+             << "desc"
+             << "</td>"   << std::endl
+             << "</tr>"   << std::endl;
+      }
+
+      regname = "DEBUG_LAST_CLUSTER";
+      for (int j = 0; j < 8; ++j) {
+        std::stringstream specregname;
+        specregname << regname << "_" << j;
+        *out << "<tr>"    << std::endl
+             << "<td>"    << std::endl
+             << specregname.str()
+             << "</td>"   << std::endl;
+
+        for (int i = 0; i < 12; ++i) {
+          INFO("GLIBMonitor::buildTriggerStatusTable creating OH" << i << " table header");
+          *out << "<td id=\"" << monpair->second.infoSpace->name() << "-OH" << i << "_" << specregname.str() << "\">" << std::endl
+               << "N/A"
+               << "</td>"   << std::endl;
+        }
+        *out << "<td>"    << std::endl
+             << "add"
+             << "</td>"   << std::endl
+             << "<td>"    << std::endl
+             << "desc"
+             << "</td>"   << std::endl
+             << "</tr>"   << std::endl;
+      }
+      break;
+    }
+  }
+  *out << "</tbody>" << std::endl
+       << "</table>" << std::endl
+       << "</div>"   << std::endl;  // closes Trigger Status tab
```

Comment:
Generic comment: this is all referring to the GLIB while being common to any kind of AMC board. Shouldn't we start moving/renaming the stuff as we are already having a large update?

Reply:
That's quite a bit more of an update, so I'll do that once the content is stabilized.